### PR TITLE
A new cheatsheet for following the StackClub git workflow

### PR DESCRIPTION
**Pull request by DouglasLeeTucker**
This code is ready for review!  Could you take a look please, @Madejski ?
Here are particular items I am hoping for review:
1. The git merge steps described in the cheatsheet.
2. In general:  anything in particular missing?

Many thanks!

### CODE REVIEW

File: GettingStarted/CheatSheet.txt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+General git tutorial stuff:
+  https://www.youtube.com/watch?v=HVsySz-h9r4
+  https://medium.com/@christo8989/what-college-students-should-learn-about-git-6bbf6eaac39c
+  https://github.com/drphilmarshall/GettingStarted
+  
+
+-----------------------------------------------------------
+Useful Slack Channels:
+  dm-lsp-users
+  dm-newbies
```
Comment (Madejski) [inline GettingStarted/CheatSheet.txt]: I don't see the "dm-newbies" channel - does a "newbie" have to request access to it?

File: GettingStarted/CheatSheet.txt [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  dm-stack? (I can't find this one. -- DLT)
+
```
Comment (Madejski) [inline GettingStarted/CheatSheet.txt]: I think there is no specific "dm-stack" channel, just the "stack club" channel at https://lsstc.slack.com/messages/C9YRAS4HM/ (I assume this is visible to anyone in the club)

**Comment by DouglasLeeTucker** [comment]
Thanks, @Madejski !  For some reason, I did not get any notifications in my inbox, so only saw your comments a bit later.  I will digest your suggestions and start making fixes.  Thanks!

**Comment by DouglasLeeTucker** [comment]
(Maybe I did not get notifications because I forgot to assign myself to the pull request.  I just assigned myself.  We'll see if that works!)

**Comment by DouglasLeeTucker** [comment]
OK, I am getting the notifications now (via e-mail).  This happened after I had assigned myself to the pull request, but I am not sure if doing this was the actual trigger for getting the notifications.  Strange...

**Comment by Madejski** [comment]
Those were all minor.  I am planning to do a more extensive "road test" of your cheat-sheet - probably on Monday.  Thanks!  --Greg

File: GettingStarted/CheatSheet.txt [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+git status
+
+
+# Push updates to origin on github:
+git push origin project/helloworld/douglasleetucker
+
+
+# Make "pull" request:
+Go to https://github.com/LSSTScienceCollaborations/StackClub/pulls and click on green "New pull request" button.
+
```
Comment (Madejski) [inline GettingStarted/CheatSheet.txt]: I think "pull request" is possibly the most mysterious command to newbies.  A couple of lines briefly explaining what this "pull request" is, and the approval process might be helpful to newbies...

File: GettingStarted/CheatSheet.txt [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+# After getting e-mail saying "pull" request has been approved, perform a "merge":
+# (Note:  there may be some redundancy here.  I was following the workflow described in
+#         the youtube video listed under "General git tutorial stuff" in the "Other useful
+#         git commands" section below.)
+git checkout master     # Switch from project/helloworld/douglasleetucker back to master branch (locally)
+git pull origin master  # Make sure local master branch is up-to-date with remote origin master branch on github
+git branch --merged     # Check to see if project/helloworld/douglasleetucker has been merged locally
+git merge project/helloworld/douglasleetucker      # Merge project/helloworld/douglasleetucker with local master branch
```
Comment (Madejski) [inline GettingStarted/CheatSheet.txt]: Perhaps a line explaining the "merge" would be valuable;  is there an approval similar to that associated with "pull request" taking place here?

**Pull request merged**